prover/src/constraints/evaluator/default.rs

```diff
@@ -39,6 +40,7 @@ pub struct DefaultConstraintEvaluator<'a, A: Air, E: FieldElement<BaseField = A:
     air: &'a A,
     boundary_constraints: BoundaryConstraints<E>,
     transition_constraints: TransitionConstraints<E>,
+    lagrange_constraints_evaluator: LagrangeConstraintsBatchEvaluator<E>,
```

Comment:
I would make this an optional field - this way, we won't have to handle optionality inside `LagrangeConstraintsBatchEvaluator`.

Reply:
done